cmds/core/wget/wget_test.go

```diff
@@ -165,6 +156,29 @@ func TestWget(t *testing.T) {
 	}
 }
 
-func TestMain(m *testing.M) {
-	testutil.Run(m, main)
+func TestDefaultOutputPath(t *testing.T) {
+	tests := []struct {
+		path   string
+		output string
```

Comment:
nit: it's common to name/prefix these with `want`, so that it's clear to someone reading/writing tests which test case fields are the values you're setting vs which are the values you're expecting.